Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier
# [BIA-906] Implements tests for new ODS 5.3 - RLS Collection.

To test this:
1. Checkout BIA-906 branch.
2. Compile the Solution.
3. Select the tests associated to the views on the RLS collection: 
     `rls_StudentDataAuthorization`
     `rls_UserAuthorization`
     `rls_UserDim`
     `rls_UserStudentDataAuthorization`
4. Execute these tests
5. All the tests should be executed successfully.
